dbms/src/Flash/Planner/PlanType.cpp

```diff
@@ -21,8 +21,6 @@ String PlanType::toString() const
 {
     switch (enum_value)
     {
-    case Source:
```

Comment:
PhysicalSource only use for DAGQueryBlock.
After DAGQueryBlock removed, it can be removed.